apps/web/src/app/(private)/_data-access/index.ts

```diff
@@ -170,3 +177,12 @@ export const getProviderLogCached = cache(async (uuid: string) => {
   const scope = new ProviderLogsRepository(workspace.id)
   return await scope.findByUuid(uuid).then((r) => r.unwrap())
 })
+
+export const getEvaluationsByDocumentUuidCached = cache(
```

Comment:
Does need to be cached? Is called more than once in a request?

Reply:
🤷🏼 doesn't hurt no?